documentation/source/quickstart.rst

```diff
@@ -250,6 +250,24 @@ writes are not applied immediately, but delayed until the next write cycle.
 Use ``sig.setimmediatevalue(new_val)`` to set a new value immediately
 (see :meth:`~cocotb.handle.ModifiableObject.setimmediatevalue`).
 
+In addition to regular value assignments (deposits), signals can be forced
+to a predetermined value or frozen at their current value. To achieve this,
+the various actions derived from :class:`~cocotb.handle.SetAction` can be used.
```

Comment:
You'll need to add `autoclass` directives to the `library_reference.rst` in order for this link to work. I suggest you instead use something like:
```
.. _set-actions-section

Set actions   # this is a lousy title, please use a better one!
===========

.. currentmodule:: cocotb.handle

.. autoclass:: Deposit
```
and then make this line
```suggestion
the various actions described in :ref:`set-actions-section` can be used.
```